package.json

```diff
@@ -13,13 +13,7 @@
     "android",
     "ios",
     "cpp",
```

Comment:
what's this for?

Reply:
I think it's auto generated by `yarn init`, payment react project also has [it](https://github.com/stripe/stripe-react-native/blob/f11f8dc6c50b184009834f6eb13d3096ce1f8a56/package.json#L16)